docs/tutorials/tutorial-sql-null.md

```diff
@@ -0,0 +1,214 @@
+The tutorial focuses on filters using the logical NOT operation on columns with NULL values.
+
+## Prerequisites
+
+Before starting this tutorial, download and run Apache Druid on your local machine as described in
+the [Local quickstart](index.md).
+
+The tutorial assumes you are familiar with using the [Query view](./tutorial-sql-query-view.md) to ingest and query data.
+
+The tutorial also assumes you have not changed any of the default settings for null handling.
+
+## Load data with null values
+
+The tutorial loads some data with null values for string and numeric columns as follows:
+
+```json
+{"date": "1/1/2024 1:02:00","title": "example_1","string_value": "some_value","numeric_value": 1}
+{"date": "1/1/2024 1:03:00","title": "example_2","string_value": "another_value","numeric_value": 2}
+{"date": "1/1/2024 1:04:00","title": "example_3","string_value": "", "numeric_value": null}
+{"date": "1/1/2024 1:05:00","title": "example_4","string_value": null, "numeric_value": null}
+```
+
+Run the following query in the Druid Console to load the data:
+
+```sql
+REPLACE INTO "null_example" OVERWRITE ALL
+WITH "ext" AS (
+  SELECT *
+  FROM TABLE(
+    EXTERN(
+      '{"type":"inline","data":"{\"date\": \"1/1/2024 1:02:00\",\"title\": \"example_1\",\"string_value\": \"some_value\",\"numeric_value\": 1}\n{\"date\": \"1/1/2024 1:03:00\",\"title\": \"example_2\",\"string_value\": \"another_value\",\"numeric_value\": 2}\n{\"date\": \"1/1/2024 1:04:00\",\"title\": \"example_3\",\"string_value\": \"\", \"numeric_value\": null}\n{\"date\": \"1/1/2024 1:05:00\",\"title\": \"example_4\",\"string_value\": null, \"numeric_value\": null}"}',
+      '{"type":"json"}'
+    )
+  ) EXTEND ("date" VARCHAR, "title" VARCHAR, "string_value" VARCHAR, "numeric_value" BIGINT)
+)
+SELECT
+  TIME_PARSE("date", 'd/M/yyyy H:mm:ss') AS "__time",
+  "title",
+  "string_value",
+  "numeric_value"
+FROM "ext"
+PARTITIONED BY DAY
+```
+
+After Druid finishes loading the data, run the following query to see the table:
+
+```sql
+SELECT * FROM "null_example"
+```
+
+|`__time`|`title`|`string_value`|`numeric_value`|
+|---|---|---|---|---|---|
+|`2024-01-01T01:02:00.000Z`|`example_1`|`some_value`|1|
+|`2024-01-01T01:03:00.000Z`|`example_2`|`another_value`|2|
+|`2024-01-01T01:04:00.000Z`|`example_3`|`empty`|`null`|
+|`2024-01-01T01:05:00.000Z`|`example_4`|`null`|`null`|
+
+Note the difference in the empty string value for example 3 and the null string value for example 4.
+
+## String query example
+
+The queries in this section illustrate null handling with strings.
+The following query filters rows where the string value is not equal to "some_value":
+
+```sql
+SELECT COUNT(*)
+FROM "null_example"
+WHERE "string_value" != 'some_value'
+```
+
+Druid returns 2 for "another_value" and the empty string "". The null value is not counted.
+
+Note that the null value is included in COUNT(*) but not as a count of the values in the column as follows:
```

Comment:
code font for `COUNT(*)` since it's a specific instance of COUNT and not COUNT as a function in general